news/5241.documentation

```diff
@@ -0,0 +1,2 @@
+Add new website IPN
```

Comment:
@sneridagh don't we have an "internal" towncrier option?

Reply:
Yes: https://6.docs.plone.org/contributing/index.html#contributing-change-log-label

Reply:
yes but that's docs, not internal imho